apis/rust/node/src/node/mod.rs

```diff
@@ -1007,3 +961,58 @@ impl DerefMut for ShmemHandle {
 
 unsafe impl Send for ShmemHandle {}
 unsafe impl Sync for ShmemHandle {}
+
+/// Init Opentelemetry Tracing
+#[cfg(feature = "tracing")]
+pub fn init_tracing(
+    node_id: &NodeId,
+    dataflow_id: &DataflowId,
+) -> eyre::Result<Arc<Mutex<Option<OtelGuard>>>> {
+    let node_id_str = node_id.to_string();
+    #[cfg(feature = "tracing")]
+    let guard: Arc<Mutex<Option<OtelGuard>>> = Arc::new(Mutex::new(None));
+    #[cfg(feature = "tracing")]
```

Comment:
These two `cfg`s should be unnecessary because the whole method is `cfg(feature = "tracing")`